core/vote/vote_manager.go

```diff
@@ -197,7 +193,8 @@ func (voteManager *VoteManager) UnderRules(header *types.Header) (bool, uint64,
 			}
 		}
 	}
-	for blockNumber := targetNumber + 1; blockNumber <= targetNumber+naturallyJustifiedDist; blockNumber++ {
+	delayAfterImportBlockForRobust := uint64(2)
```

Comment:
Can you put some comments on `delayAfterImportBlockForRobust`, I am confused about it.

Reply:
// we do the following check when importing new blocks:
	// 	chainHeight - height(new imported block) <= upperLimitOfVoteBlockNumber
	// but between importing that block and calling func UnderRules, chainHeight may have extended
	// so check 2 more blocks here for robust. 
	
	It's possible?